index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Hello Microverse</title>
+</head>
+<body>
+  <p>Hello Microverse</p>
```

Comment:
Please change the paragraph Hello Microverse to an h1 header because it was one of the requirements! 💪

Reply:
Thankyou for the review, implemented all the required changes